apps/mobile/package.json

```diff
@@ -36,6 +36,7 @@
   "dependencies": {
     "@crowdin/ota-client": "2.0.0",
     "@expo/vector-icons": "14.0.0",
+    "@gorhom/bottom-sheet": "4.6.3",
```

Comment:
I moved `@gorhom/bottom-sheet` dependancy into the UI library. 

Is there some reason not to use standard `Sheet`? Maybe we can export `BottomSheetScrollView` from the UI library and import here to avoid extra dependancies?

The same for `prism-react-renderer`, if we can move that component into the library then we could save the import here